future-apps/payroll/package.json

```diff
@@ -1,39 +1,45 @@
 {
   "name": "@aragon/future-apps-payroll",
-  "version": "1.0.0",
-  "main": "index.js",
-  "scripts": {
-    "test": "TRUFFLE_TEST=true npm run ganache-cli:test",
-    "test:gas": "GAS_REPORTER=true npm test",
-    "lint": "solium --dir ./contracts",
-    "coverage": "SOLIDITY_COVERAGE=true npm run ganache-cli:test",
-    "ganache-cli:test": "./node_modules/@aragon/test-helpers/ganache-cli.sh"
-  },
-  "keywords": [],
+  "version": "0.0.1",
+  "description": "",
+  "keywords": [
+    "app",
+    "aragon",
+    "dao",
+    "payroll"
+  ],
+  "license": "(GPL-3.0-or-later OR AGPL-3.0-or-later)",
   "author": "Aragon Institution MTU <contact@aragon.one>",
   "contributors": [
-    "ßingen <bingen@aragon.one>"
+    "ßingen <bingen@aragon.one>",
+    "Lisandro Corbalán <lisandro.corbalan@altoros.com>",
+    "Sebastián Galiano <sebastian.galiano@altoros.com>"
   ],
-  "license": "GPL-3.0",
-  "description": "",
-  "devDependencies": {
-    "@aragon/apps-vault": "^2.0.1",
-    "@aragon/test-helpers": "^1.0.1",
-    "babel-helpers": "^6.24.1",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-stage-2": "^6.24.1",
-    "babel-preset-stage-3": "^6.17.0",
-    "babel-register": "^6.26.0",
-    "ganache-cli": "^6.1.0",
-    "solidity-coverage": "0.3.5",
-    "solium": "^1.1.3",
-    "truffle": "4.0.5",
-    "truffle-hdwallet-provider": "0.0.3",
-    "webpack": "^3.7.1"
+  "scripts": {
+    "build": "cd app && npm run build",
+    "coverage": "SOLIDITY_COVERAGE=true npm test",
+    "dev": "aragon run --network rpc --http localhost:8001 --http-served-from ./app/dist",
+    "dev:reset": "npm run dev -- --reset",
+    "postinstall": "cd app && npm install",
+    "lint": "solium --dir ./contracts",
+    "start": "aragon run --network rpc --kit PayrollKit --kit-init @ARAGON_ENS",
+    "test": "TRUFFLE_TEST=true ./node_modules/@aragon/os/scripts/ganache-cli.sh",
+    "test:gas": "GAS_REPORTER=true npm test"
   },
   "dependencies": {
-    "@aragon/apps-finance": "^1.0.0",
-    "@aragon/os": "^3.0.6"
+    "@aragon/apps-finance": "^2.0.0-beta.2",
+    "@aragon/os": "^4.0.0-beta.3",
+    "@aragon/ppf-contracts": "^1.0.2"
+  },
+  "devDependencies": {
+    "@aragon/apps-shared-migrations": "^1.0.0",
+    "@aragon/apps-shared-minime": "^1.0.0",
+    "@aragon/apps-vault": "^3.0.0-beta.2",
+    "@aragon/cli": "git+https://github.com/aragon/aragon-cli.git#06e2f1cee4384549441d7ec8807b7635dbd64b69",
```

Comment:
We had to use the latest version from repo because the version published in npm is broken.

Reply:
Aragon CLI was updated to the just-released version (v5.1.0)